README.md

```diff
@@ -27,6 +27,10 @@ To get a local copy up and running follow these simple example steps.
 - GitHub: [@barakadan](https://github.com/barakadanny)
 - LinkedIn: [danny baraka](https://www.linkedin.com/in/danny-baraka-589156169/)
 
+👤 **George William Kidde**
+  - Github: [@kidde60](https://github.com/kidde60)
+  - Twitter: [@KGWilliam60](https://twitter.com/KGWilliam60)
+  - Linkedin: [Linkedin](https://www.linkedin.com/in/kidde-george-william/)
 
 ## 🤝 Contributing
 
```

Comment:
- [ ] note I'm reviewing here cause the file doesn't available here 

- [ ] Your code is clean however you should separate the code that asks for the user's answer and the one that validates if it's  `yes or no or 1 or 2 ....etc` into private methods and used them in your methods to create the book, person, ...etc in order to achieve the `single responsibility principle` please make sure to do that in all the methods 
<img width="633" alt="Screenshot 2022-11-21 at 14 20 15" src="https://user-images.githubusercontent.com/61790587/203066466-8267b202-003c-41d3-8c16-dade7e9dc792.png">